dev/TeachingTip/TeachingTip.cpp

```diff
@@ -710,28 +720,46 @@ void TeachingTip::OnIsOpenChanged()
             this->ApplyTemplate();
         }
```

Comment:
Just noticed this -- this seems super dubious. Why are we trying to force layout-related things to happen outside of layout?

Reply:
In the scenario where the tip was defined in a resource dictionary we will never apply the teaching tip template, which means we don't have access to the template part that we want to place into the popup we are creating

---
In reply to: [268777304](https://github.com/Microsoft/microsoft-ui-xaml/pull/480#discussion_r268777304) [](ancestors = 268777304)